src/django_browser_reload/views.py

```diff
@@ -41,6 +43,17 @@
 # Communicate template changes to the running polls thread
 should_reload_event = threading.Event()
 
+# Communicate the connection should close
+should_exit_event = threading.Event()
+
+
+def handle_sigint(signum: int, frame: Any) -> None:
+    should_exit_event.set()
+    sys.exit(0)
```

Comment:
Without adding a sys.exit CONTROL+C wasn't exiting, but I don't fully understand why. Does listening to SIGINT here prevent SIGTERM from running over at https://github.com/django/django/blob/d4a2809c2b878317a4fa0e1cbaf07322ffd7f2fb/django/utils/autoreload.py#L663 🤔